definitions/npm/lodash_v4.x.x/flow_v0.38.x-v0.39.x/lodash_v4.x.x.js

```diff
@@ -230,20 +230,20 @@ declare module 'lodash' {
     now(): number;
 
     // Function
-    after(n: number, fn: Function): Function;
+    after<R>(n: number, fn: (...args: Array<any>) => R): (...args: Array<any>) => R | void;
```

Comment:
The annotations for flow v.40+ are a bit fancier. As far as I can tell, all we can express here is that the function returned by `after` returns `fn`'s return type, or `void`; we can't express that its arguments should be exactly the same as `fn`'s.